eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -159,10 +159,34 @@ jobs:
 
       - template: ../steps/upload-repository-on-failure.yml
 
+      - task: Maven@3
+        displayName: 'Generate aggregate code coverage report'
+        condition: and(eq(variables['OsName'], 'Windows'), eq(variables['JavaTestVersion'], '1.11'), eq('${{ parameters.ServiceDirectory }}', 'communication'))
+        inputs:
+          mavenPomFile: sdk/${{ parameters.ServiceDirectory }}/pom.xml
+          options: -Pcoverage
+          mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
+          javaHomeOption: 'JDKVersion'
+          jdkVersionOption: $(JavaTestVersion)
+          jdkArchitectureOption: 'x64'
+          goals: jacoco:report-aggregate
+          # we want to run this when TestFromSource isn't true
+          condition: and(succeeded(), ne(variables['TestFromSource'],'true'))
+
       - task: PublishTestResults@2
         condition: always()
         inputs:
           mergeTestResults: true
           testRunTitle: 'Live tests for ${{ parameters.ServiceDirectory }} $(DisplayName)'
           ${{ if ne(parameters.TestResultsFiles, '') }}:
             testResultsFiles: ${{ parameters.TestResultsFiles }}
+
+      # Azure DevOps only seems to respect the last code coverage result published, so only do this for Windows + Java LTS.
+      # Code coverage reporting is setup only for Track 2 modules.
+      - task: PublishCodeCoverageResults@1
+        condition: and(eq(variables['OsName'], 'Windows'), eq(variables['JavaTestVersion'], '1.11'))
```

Comment:
Add this condition here as well - `, eq('${{ parameters.ServiceDirectory }}', 'communication')`

Reply:
Done, thanks Srikanta